src/api.js

```diff
@@ -0,0 +1,13 @@
+const BASE_URL = 'https://mate-api.herokuapp.com';
+
+export function getTodos() {
+  return fetch(`${BASE_URL}/todos`)
+    .then(response => response.json())
+    .then(result => result.data);
+}
+
+export function getUser(userId) {
+  return fetch(`${BASE_URL}/users/${userId}`)
+    .then(response => response.json())
+    .then(response => response.data);
```

Comment:
Don't repeat yourself
You could create one variable 'request' for both cases

Reply:
Agree, good comment.